src/commands/createNewProject/CSharpProjectCreator.ts

```diff
@@ -129,32 +128,34 @@ export class CSharpProjectCreator implements IProjectCreator {
     }
 
     public getRecommendedExtensions(): string[] {
-        return ['ms-vscode.csharp'];
+        return super.getRecommendedExtensions().concat(['ms-vscode.csharp']);
     }
 
     /**
      * Validates the project has the minimum Functions SDK version that works on all OS's
      * See this bug for more info: https://github.com/Microsoft/vscode-azurefunctions/issues/164
      */
     private async validateFuncSdkVersion(csprojPath: string, csprojContents: string): Promise<void> {
-        try {
-            const minVersion: string = '1.0.8';
-            const lineMatches: RegExpMatchArray | null = /^.*Microsoft\.NET\.Sdk\.Functions.*$/gm.exec(csprojContents);
-            if (lineMatches !== null && lineMatches.length > 0) {
-                const line: string = lineMatches[0];
-                const versionMatches: RegExpMatchArray | null = /Version=(?:"([^"]+)"|'([^']+)')/g.exec(line);
-                if (versionMatches !== null && versionMatches.length > 2) {
-                    const version: SemVer = new SemVer(versionMatches[1] || versionMatches[2]);
-                    this._telemetryProperties.cSharpFuncSdkVersion = version.raw;
-                    if (version.compare(minVersion) < 0) {
-                        const newContents: string = csprojContents.replace(line, line.replace(version.raw, minVersion));
-                        await fse.writeFile(csprojPath, newContents);
+        if (!isWindows) {
```

Comment:
Does this function fail silently if it's a Windows OS or is this validation not necessary?

Reply:
Validation isn't necessary on Windows - 1.0.8 is only required for Mac and Linux